src/components/encounter/ohri-encounter-form.component.tsx

```diff
@@ -544,6 +553,42 @@ export const OHRIEncounterForm: React.FC<OHRIEncounterFormProps> = ({
         return Promise.all(saveAttachmentPromises).then(() => response);
       });
     }
+
+    if (encounterForSubmission.obs?.length || encounterForSubmission.orders?.length) {
+      if (Object.keys(encounterContext?.patientIdentifier).length) {
+        return saveIdentifier(patient, encounterContext.patientIdentifier)
+          .then(() => saveEncounterWithAttachments(encounterForSubmission))
+          .catch((error) => {
```

Comment:
At this point; do we want to change to move to: `isSubmitting: false`?

Reply:
Makes sense